test/integration/wyatt_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class WyattTest < ActionDispatch::IntegrationTest
+  test "shows specific message when visiting '/wyatt'" do
+    get "/wyatt"
+    assert_response :success
+
+    assert_select "p", "Hi James!"
```

Comment:
If the test runs and the request does NOT include a paragraph `<p>` tag with the text 'Hi James!' then the test will fail.